src/components/funFacts/FunFactComponent.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import { funFacts, FunFact } from "./funfacts"
+import { useState } from "react"
+import "./FunFactPage.scss"
+
+interface FunFactsProps {
+    funFacts: FunFact[];
+}
+
+export const FunFactComponent: React.FunctionComponent<FunFactsProps> = ({ funFacts }) => {
+
+  const [selectedFact, setSelectedFact] = useState<number>(0);
+
+  function toggleFactDetail(funFactId:number){
+    if (selectedFact===funFactId){
```

Comment:
1. Prefer arrow functions
2. Spaces around certain things
```suggestion
  const toggleFactDetail = (funFactId: number) => {
    if (selectedFact === funFactId){
```

Reply:
Done, thanks